components/dot_menu/dot_menu.tsx

```diff
@@ -430,6 +431,7 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
             <MenuWrapper
                 open={this.props.isMenuOpen}
                 onToggle={this.handleDropdownOpened}
+                className={'dotmenu-dropdown-menu'}
```

Comment:
according to the comment in `dot_menu.scss`:

```suggestion
                className={'dropdown-menu__dotmenu'}
```